### PR TITLE
SONAR-14647 added licence information

### PR DESCRIPTION
**Pull request by tobias-trabelsi-sonarsource**
as stated in [SONAR-14647](https://jira.sonarsource.com/browse/SONAR-14647)

### CODE REVIEW

File: sonarqube/license.md [hunk 1]
```diff
@@ -1 +1 @@
-View [license information](http://www.gnu.org/licenses/lgpl.txt) for the software contained in this image.
+SonarQube Community Edition is licensed under [GNU Lesser General Public License, Version 3.0]((http://www.gnu.org/licenses/lgpl.txt)). SonarQube Developer, Enterprise and Data Center Editions are licensed under [SonarSource Terms and Conditions](https://www.sonarsource.com/docs/sonarsource_terms_and_conditions.pdf)
```
Comment (MikeBirnstiehl) [inline sonarqube/license.md]:
```suggestion
SonarQube Community Edition is licensed under [GNU Lesser General Public License, Version 3.0]((http://www.gnu.org/licenses/lgpl.txt)). SonarQube Developer, Enterprise, and Data Center Editions are licensed under [SonarSource Terms and Conditions](https://www.sonarsource.com/docs/sonarsource_terms_and_conditions.pdf).
```

**Review by MikeBirnstiehl — APPROVED** [review]
Just some punctuation, other than that LGTM.

File: sonarqube/license.md (hunk 1 above)
Comment (tobias-trabelsi-sonarsource) [reply sonarqube/license.md]: added to the PR, thanks for helping with my grammar :+1:

**Review by christophelevis — APPROVED** [review]
:+1

**Pull request merged**